src/theme/components/notification.ts

```diff
@@ -13,7 +13,7 @@ const Notification = {
     return {
       bg,
       display: "flex",
-      fontSize: "-1", // slightly smaller than the default size
+      fontSize: "text.caption", // slightly smaller than the default size
```

Comment:
Do you still need the comment?

Reply:
Yes, let’s keep the comment to show that the `fontSize` value was intentional.